cli/dcoscli/package/main.py

```diff
@@ -334,6 +334,19 @@ def _install(package_name, package_version, options_path, app_id, cli,
     pkg = package_manager.get_package_version(package_name, package_version)
 
     pkg_json = pkg.package_json()
+
+    selected = pkg_json.get('selected')
+    if selected:
+        link = ('https://mesosphere.com/'
+                'catalog-terms-conditions/#certified-services')
+    else:
+        link = ('https://mesosphere.com/'
+                'catalog-terms-conditions/#community-services')
+    emitter.publish(
+        ('By Deploying, you agree to '
+         'the Terms and Conditions ' + link + '\n')
+    )
+
```

Comment:
We should also always prompt the user and print "Continue installing?".

Line 353 needs to execute irrespective of line 351.